src/cli/rustup_mode.rs

```diff
@@ -819,20 +819,18 @@ async fn check_updates(cfg: &Cfg<'_>, opts: CheckOpts) -> Result<ExitCode> {
         Some(_) | None => channels_len,
     };
 
-    let bold = if use_colors {
-        Style::new().bold()
+    let (bold, transient, error, good, warn) = if use_colors {
+        (Style::new().bold(), TRANSIENT, ERROR, GOOD, WARN)
     } else {
-        Style::new()
+        (
```

Comment:
Nit: Actually this can be simplified as `Default::default()` :)

Reply:
Huh, I had assumed `Default` wasn't implemented for tuples.